composer.json

```diff
@@ -19,7 +19,7 @@
     "ext-pdo": "*",
     "fideloper/proxy": "^4.2",
     "fruitcake/laravel-cors": "^2.0",
-    "guzzlehttp/guzzle": "^7.0.1",
+    "guzzlehttp/guzzle": "^7.0",
```

Comment:
Why did you change that?

Reply:
Sorry, I think when I was installing something for email testing and new depency downgrade **guzzlehttp**. I fixed that.